metrics/web/lib/common/presentation/metrics_theme/model/dark_metrics_theme_data.dart

```diff
@@ -201,6 +201,7 @@ class DarkMetricsThemeData extends MetricsThemeData {
             buttonAttentionLevel: MetricsButtonAttentionLevel(
               positive: MetricsButtonStyle(
                 color: ColorConfig.primaryColor,
+                elevation: 0.0,
```

Comment:
Should we just make elevation default value `0` in MetricsButtonStyle? As by default, I think elevation is 0 and most likely will be 0 for all of the cases. In that case, we can remove all the places where you explicitly say it's 0.0. It might be also worth considering removing elevation from MetricsButtonStyle constructor - is most likely we'll not have elevation here at all.